DS-5881 by serhiy-chornobay: Update package.json for Social blue theme

Crazylord opened the pull request:
## Problem
Update package.json library

## Issue tracker
- https://jira.goalgorilla.com/browse/DS-5881

## HTT
- [ ] Check out the code changes
- [ ] Checkout to this branch

## Release notes
Need remove node_modules fosder and run npm install

Kingdutch commented:
Hi @Crazylord,

I see this PR introduces a `package-lock.json`. We use [Yarn](https://yarnpkg.com/en/) in our projects to maintain the dependencies. Please use Yarn to update the `yarn.lock` file and remove the `package-lock.json` file.

Could you also add some text explaining the changes you've made to Gulp (explain why)? I see that some of the changes change the paths of files which makes them relative to the working directory of the script instead of the location of the gulpfile (which is a constant).

~ Alexander

Crazylord commented:
Hi @Kingdutch .
I made **npm audit** for **package.json** and check, what need update. Then updated all node modules, what recomended this audit. 
About the paths - after update, the **watch** task doesn't work with old formats of path, so I changed it.

Remove `package-lock.json`  file. Updated `yarn.lock` file

jaapjan commented:
Restarted the tests again, seems false negatives. 🤞

jaapjan commented:
Closing this one as there are all kind of issues with merging and cherry-picking. Must not be that hard to solve, but created follow-up issue for that:
https://www.drupal.org/project/social/issues/3015059

Please create new PR when working on that.
